### PR TITLE
Add v0.5.1 release notes from AutoMM

### PR DESCRIPTION
**Pull request by bryanyzhu**
*Issue #, if available:*
NA

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by gradientsky — APPROVED** [review]

File: docs/whats_new/v0.5.1.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- Improved efficient finetuning
+  - Support “bit_fit”, “norm_fit“, “lora”, “lora_bias”, “lora_norm”. In four multilingual datasets ([xnli](https://huggingface.co/datasets/xnli), [stsb_multi_mt](http://stsb_multi_mt/), [paws-x](https://huggingface.co/datasets/paws-x), [amazon_reviews_multi](https://huggingface.co/datasets/amazon_reviews_multi)), “lora_bias”, which is a combination of [LoRA](https://arxiv.org/abs/2106.09685) and [BitFit](https://arxiv.org/abs/2106.10199), achieved the best overall performance. Compared to finetuning the whole network, “lora_bias” will only finetune **<0.5%** of the network parameters and can achieve comparable performance on “stsb_multi_mt” (#1780, #1809). @Raldir @zhiqiangdon
+  - Support finetuning the [mT5-XL](https://huggingface.co/google/mt5-xl) model that has 1.7B parameters on a single NVIDIA G4 GPU. In AutoMM, we only use the T5-encoder (1.7B parameters) like [Sentence-T5](https://aclanthology.org/2022.findings-acl.146.pdf). (#1933) @sxjscience
+
+- Added more data augmentation techniques
+  - [Mixup](https://arxiv.org/pdf/1710.09412.pdf) for image data. (#1730) @sxjscience
```
Comment (zhiqiangdon) [inline docs/whats_new/v0.5.1.md:19]: @Linuxdex here?

File: docs/whats_new/v0.5.1.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  - [TrivialAugment](https://arxiv.org/pdf/2103.10158.pdf) for both image and text data. (#1792) @lzcemma
+  - [Easy text augmentations](https://arxiv.org/pdf/1901.11196.pdf). (#1756) @lzcemma
+
+- Enhanced teacher-student model distillation
+  - Support distilling the knowledge from a unimodal/multimodal teacher model to a student model. (#1670, #1895) @zhiqiangdon
+
+### More tutorials and examples
+
+- [Beginner tutorials](https://auto.gluon.ai/stable/tutorials/multimodal/index.html) of applying AutoMM to image, text, or multimodal (including tabular) data. (#1861, #1908, #1858, #1869) @bryanyzhu @sxjscience
```
Comment (zhiqiangdon) [inline docs/whats_new/v0.5.1.md:28]: @zhiqiangdon for PR #1861

**Comment by szha** [comment]
Job PR-1964-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1964/1/index.html

**Comment by szha** [comment]
Job PR-1964-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1964/2/index.html

File: docs/whats_new/v0.5.1.md (hunk 1 above)
Comment (bryanyzhu) [reply docs/whats_new/v0.5.1.md:19]: Thanks

File: docs/whats_new/v0.5.1.md (hunk 2 above)
Comment (bryanyzhu) [reply docs/whats_new/v0.5.1.md:28]: Thanks, fixed

**Review by zhiqiangdon — APPROVED** [review]
Thanks!

**Comment by szha** [comment]
Job PR-1964-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1964/3/index.html

**Pull request merged**